handbook/sales/onboarding/joining_customer_calls.md

```diff
@@ -0,0 +1,25 @@
+# Joining customer calls
+
+Joining customer calls is a valuable part of any new team member's onboarding or ongoing work. It helps you quickly learn what customers are directly interested in, and how we communicate the product to them.
+
+## Types of calls
+
+While calendar events won't have one of these exact lables, knowing the types of calls will help you categorize calls so you can join the ones you'll find most helpful. 
+
+1. **New prospect:** the first call with a prospect, who may not know much about Sourcegraph. Often named "Intro Call."
+1. **Installation:** to walk through setting up Sourcegraph with a technical point of contact.  
+1. **Trial check-in:** to discuss how things are going during a trial.
+1. **Training session:** a demo walkthrough of common use cases, an advanced tutorial, or a Q&A, all a customer's end users. Often have names like "Sourcegraph 101," "Webinar,"" "Demo," "Q&A.""
+1. **Existing customer check-in:** to hear how things might be improved or learn about exciting ways a customer is using Sourcegraph. 
```

Comment:
```suggestion
1. **Existing customer check-in:** to hear how things might be improved or learn about exciting ways a customer is using Sourcegraph. These can vary significantly, so it can be worthwhile to check in with the team to understand context in advance.
```